src/ocm/components/clusterConfiguration/networkConfiguration/NetworkConfigurationForm.tsx

```diff
@@ -58,6 +58,10 @@ const NetworkConfigurationForm: React.FC<{
   const isAutoSaveRunning = useFormikAutoSave();
   const errorFields = getFormikErrorFields(errors, touched);
 
+  // DHCP allocation is currently not supported with Nutanix
+  // https://issues.redhat.com/browse/MGMT-12382
+  const isVipDhcpAllocationDisabled = cluster.platform?.type === 'nutanix';
```

Comment:
Should we maybe not disable it in case we have a Nutanix cluster which has DCHP ON for whatever reason (eg by using the API previously)?

That would let the user switch it OFF and then set the IPs.

Reply:
I think it would be better if set DHCP allocation off in that case. Because with Nutanix, the auto allocation will fail.

If the IPs are already allocated, they should stay (in the input fields). But I think the checkbox should be hidden anyway.

Reply:
Makes sense :+1: